python/paddle/vision/transforms/functional.py

```diff
@@ -36,11 +36,6 @@
 from . import functional_cv2 as F_cv2
 from . import functional_tensor as F_t
 
-__all__ = [
```

Comment:
structure it like
vision.transforms
└── vision.transforms.functional

Reply:
done